geom/src/tessellation.rs

```diff
@@ -180,9 +205,44 @@ impl Tessellation {
         }
         result
     }
+
+    /// Produces a GeoJSON multipolygon consisting of individual triangles. Optionally map the
```

Comment:
I was hoping to get rid of this silliness, but `osm2streets` currently draws arrow outlines via GeoJSON. So until making `to_outline` return a real polygon, this stays

Reply:
You've probably already thought about this, but maybe it makes sense to not tessellate the line string arrow...

Like maybe:

```
fn make_outline_arrow(thickness, cap) -> geom::Polygon {
    let exterior = make_arrow(thickness * 2, cap);
    let interior = make_arrow(thickness / 2, cap );
    Polygon::with_holes(exterior, interior)
}
```

Reply:
... What a good idea! It never occurred to me, this is exactly why reviews are awesome. :)

![arrow1](https://user-images.githubusercontent.com/1664407/189633049-d9fa8eb3-9e3a-404e-a7e9-c03b747bbd59.png)
![arrow2](https://user-images.githubusercontent.com/1664407/189633061-1d8d2443-a7ac-4a3c-b219-730885d20acd.png)

I haven't quite replicated the same nice outline style with this method though. The polyline has to be trimmed back partly for the arrowhead to appear correctly. I'll keep playing with it, and remove this technical debt of geojson triangles once it works, or once `to_outline` can always produce nice rings.

Reply:
Or maybe one day [the revolution](https://github.com/georust/geo/issues/641) will finally come.